source/UIOverlay.cpp

```diff
@@ -0,0 +1,31 @@
+#include "UIOverlay.hpp"
+
+void UIOverlay::setOverlay(Ogre::Overlay *ov) {
+
+	overlay = ov;
+}
+
+Ogre::Overlay *UIOverlay::getOverlay(void) const {
+
+	return overlay;
+}
+
+Ogre::Vector2 UIOverlay::pixelsToRelative(Ogre::Vector2 pixels) {
+
+	Ogre::Real x;
+	Ogre::Real y;
```

Comment:
Avoid uninitialized variables unless initialising them would cause some performance loss.
(in this case x & y could even be inlined)